mirgecom/euler.py

```diff
@@ -34,6 +34,12 @@
 
 .. autofunction:: get_inviscid_timestep
 .. autofunction:: get_inviscid_cfl
+
+References
+----------
+.. [DGBook] Hesthaven and Warburton (2008), Nodal DG Methods, Springer
```

Comment:
Start a convention for naming those. `hesthaven_2008`, mabye? (Document that in a comment where you put the references.)